torch/_torch_docs.py

```diff
@@ -7833,6 +7835,9 @@ def merge_dicts(*dicts):
 .. note:: When :attr:`compute_uv` = ``False``, backward cannot be performed since `U` and `V`
           from the forward pass is required for the backward operation.
 
+.. note:: With the complex-valued input the backward operation works correctly only
+          for gauge invariant loss functions.
```

Comment:
nit: could you add a reference link that explains what "gauge invatiant loss functions" is here?

Reply:
I added a link to https://re-ra.xyz/Gauge-Problem-in-Automatic-Differentiation/